lib/daru/io/importers/excel.rb

```diff
@@ -45,22 +55,26 @@ class Excel < Base
         #   #    3        4    Franz      nil    Paris      nil
         #   #    4        5   George      5.5     Tome    a,b,c
         #   #    5        6  Fernand      nil      nil      nil
-        def initialize(path, worksheet_id: 0)
+        def initialize(path, sheet: 0, headers: true)
           optional_gem 'spreadsheet', '~> 1.1.1'
```

Comment:
> Is it possible to have the optional_gem method run at the time of the gem require? Or how about "trying" to require all gems at gem require first and then throw warnings only when the specific method is called?

@v0dro To be honest, I am strictly against it. `optional_gem` is not effect-less, it outputs "Hey, we need this and that gem to do what you want". Imagine daru-io v.2.3.4, which somebody wants to use to just import some CSV? It will output two full screens of `please install <this irrelevant stuff>` just for nothing.

I believe, calling `optional_gem` _exactly_ at the moment when somebody tries to really import/export `xlsx` is a good idea.

Side note: In future, there can exist code like 
```ruby
format = controller.params[:format]
dataframe = Daru.from(format, controller.params[:input])
```
...which could probably fail too late (e.g. works for 1000 users, and fails for 1001st, because format was `dbf`, and dependency is absent), but it is another story (about APIs where `format` is a variable, and then about APIs of "early check" that we support this and that format, and probably explicitly forbid those and those).